### PR TITLE
Add withHostname option

### PR DESCRIPTION
**Pull request by damian-kociuba**
## Description
I need option withHostname in order to distinguish which node of application fired the log. I've copied and adjusted the code from le_node repository 

## Testing
Set `withHostname: true` option and add some logs. In log view you should see the host name at the begging of string or as extra property for json

### CODE REVIEW

**Review by rjacobs-r7 — APPROVED** [review]

File: src/logger.js [hunk 1]
```diff
@@ -305,6 +308,12 @@ class Logger extends Writable {
         modifiedLog[safeLevel] = lvlName;
       }
 
+      //  Set level if present
+      if (this.withHostname) {
+        safeHost = getSafeProp(modifiedLog, 'host');
```
Comment (sbialkowski-r7) [inline src/logger.js:313]: might be nice to keep the naming the same everywhere
```suggestion
        safeHost = getSafeProp(modifiedLog, 'hostname');
```

File: test/test.js [hunk 2]
```diff
@@ -495,6 +510,44 @@ tape('Non-JSON logs may carry timestamp.', function (t) {
...
+  t.timeoutAfter(2000);
+
+  mockTest(function (data) {
+    t.true(pattern.test(data), 'matched');
+
+  });
+  const os = require('os');
+  const lvl = defaults.levels[3];
+  const tkn = token;
+  const pattern = new RegExp('^' + token +' ' + os.hostname() + ' \\w+ test\\n$'
```
Comment (sbialkowski-r7) [inline test/test.js:524]: would this be more readable with template strings?
```suggestion
  const pattern = new RegExp(`^${token} ${os.hostname()} \\w+ test\\n$`;
```

File: test/test.js [hunk 3]
```diff
@@ -495,6 +510,44 @@ tape('Non-JSON logs may carry timestamp.', function (t) {
...
+});
+
+
+tape('JSON logs may carry Hostname.', function (t) {
+  t.plan(1);
+  t.timeoutAfter(2000);
+
+  mockTest(function (data) {
+    const log = JSON.parse(data.substr(37));
+    t.true(log.host, 'has property');
```
Comment (sbialkowski-r7) [inline test/test.js:539]: can we assert the hostname here?

```suggestion
    t.equals(os.hostname(), log.host);
```

**Comment by damian-kociuba** [comment]
Thanks for the review 👌 . I've applied all remarks.

**Review by sbialkowski-r7 — APPROVED** [review]

**Comment by sbialkowski-r7** [comment]
@damian-kociuba Thanks a mill for the contribution!

This PR should be released now as `3.1.0`, give me a shout if you've any issues!

**Pull request merged**